atom/renderer/lib/inspector.coffee

```diff
@@ -1,4 +1,5 @@
 window.onload = ->
+  # this is a major security hole I think if it's accessible on the page
```

Comment:
I need to test to make sure the inspector is only available in the dev tools and not the page. If the page can access it then it's definitely something we need to worry about